server/controllers/MentorController.js

```diff
@@ -0,0 +1,41 @@
+import pool from '../db.js';
+import bcrypt from 'bcrypt';
+
+
+export default {
+    async allMentors(req,res) {
+        try {
+           const mentors = await pool.query('SELECT * FROM mentors');
+           res.json({mentors: mentors.rows})
+
+        } catch (error) {
+            res.status(500).json({error:error.message})
+        }
+    },
+
+    async createMentor(req, res) {
+        try {
+           const hashedPassword = await bcrypt.hash(req.body.password, 10);
+           const newMentor = await pool.query("INSERT INTO mentors ( firstName, lastName, email,password,address,bio,occupation,expertise) VALUES ($1, $2,$3,$4,$5,$6,$7,$8) RETURNING *", [req.body.firstName, req.body.lastName, req.body.email, hashedPassword, req.body.address, req.body.bio, req.body.occupation, req.body.expertise]) 
+           res.json({mentor: newMentor.rows[0]})
```

Comment:
Follow the specifications given in the document to do this. Check the specification for the payload

Reply:
Create a responseFormat helper function that will help you to keep your response in a standard format based on the specification